src/providers/slot/slot.ts

```diff
@@ -39,6 +39,47 @@ export class SlotProvider {
     });
   }
 
+  /**
+   * Extending the journal with empty days where there was no slots defined in the next 7 days
+   * @param slots Journal slots
+   * @returns Slots with additional empty days 
+   */
+  extendWithEmptyDays = (slots: {[k: string]: SlotItem[]}): {[k: string]: SlotItem[]} => {
+    const emptyDays = {
```

Comment:
Think the intention is to have this 7 day value configurable. In which case we should have a function that reads from config and generates these empty days.

Reply:
Good point I moved to the value to environment variables which will be provided by the app config provider. But it's not a very good pace for it I think. Any other idea?